core/DataTable/Row.php

```diff
@@ -579,6 +579,10 @@ public function sumRowMetadata($rowToSum, $aggregationOperations = array())
                         continue;
                     }
 
+                    if ($column === 'ts_archived') {
```

Comment:
I don't think this will fix the referenced issue. Logically `$column` can only be `ts_archived` at that point if `$aggregationOperations` would contain `ts_archived`. As this metadata was newly added, and a aggregation method is not set anywhere for it, I don't think this is the reason. But the `_metadata` column seems to be summed up with the normal columns it seems. Removing that might breaks something so instead I guess adding the column here might fix the issue:
https://github.com/matomo-org/matomo/blob/c75aa0a67379465bede34013661df993630e4eef/core/DataTable/Row.php#L37-L40

Reply:
Thanks @sgiehl, I figured there was some code attempting to sum up all row metadata values rather than ts_archived specifically, but wasn't sure where.